tests/test_run.py

```diff
@@ -342,3 +347,44 @@ def test_Chiral_right(check_run, system_Chiral_left, system_Chiral_right, compar
         data2 = results[1].results[key].dataSmooth
         precision = max(np.max(abs(data1)) / 1E12, 1E-11)
         assert data1 == pytest.approx(sign * data2, abs=precision), key
+
+
+def test_Te_ASE(check_run, system_Te_ASE, compare_any_result):
+    param = {'Efermi': Efermi_Te_gpaw, "tetra": True}
+    calculators = {k: v(**param) for k, v in calculators_Te.items()}
+    check_run(
+        system_Te_ASE,
+        calculators,
+        fout_name="berry_Te_ASE",
+        suffix="",
+        suffix_ref="",
+        use_symmetry=True,
+        parameters_K={
+            '_FF_antisym': True,
+            '_CCab_antisym': True
+        },
+    )
+
+
+def test_Te_ASE_wcc(check_run, system_Te_ASE_wcc, compare_any_result):
+    param = {'Efermi': Efermi_Te_gpaw, "tetra": True}
+    calculators = {}
+    for k, v in calculators_Te.items():
+        par = {}
+        par.update(param)
+        if k not in ["dos", "cumdos"]:
+            par["kwargs_formula"] = {"external_terms": False}
+        calculators[k] = v(**par)
+
+    check_run(
+        system_Te_ASE_wcc,
+        calculators,
+        fout_name="berry_Te_ASE",
+        suffix="wcc",
+        suffix_ref="",
+        use_symmetry=True,
+        parameters_K={
+            '_FF_antisym': True,
+            '_CCab_antisym': True
```

Comment:
Are you sure this parameter really work?
Last week I find in data_K, the arrays function `CCab_R` are miss match when the parameter is True.
`CC_R : [self.num_wann, self.num_wann, self.system.nRvec, 3]`
`CCab_R : [self.num_wann, self.num_wann, self.system.nRvec, 3, 3]`
So I think it shouldn't work.
```
    def CCab_R(self):
        if self._CCab_antisym:
            CCab = np.zeros((self.num_wann, self.num_wann, self.system.nRvec, 3, 3), dtype=complex)
            CCab[:, :, :, alpha_A, beta_A] = -0.5j * self.CC_R
            CCab[:, :, :, beta_A, alpha_A] = 0.5j * self.CC_R
            return CCab
        else:
            return self.system.CCab_R * self.expdK[None, None, :, None, None]
```